src/components/Sponsors/Sponsors.jsx

```diff
@@ -1,41 +1,61 @@
-// Import External Dependencies
 import Link from '../Link/Link';
-
-// Imports for Ad Content
 import AGLogo from '../../assets/ag-grid-logo.png';
+import AGChartsLogo from '../../assets/ag-charts-logo.png';
 import WebpackIcon from '../../assets/icon-square-small.svg';
-
-// Load Styling
 import '../Sponsors/Sponsors.scss';
 
-// Create and export the component
 const Sponsors = () => {
   return (
     <div className="sponsors">
-      <Link
-        className="sponsors__content"
-        to="https://www.ag-grid.com/?utm_source=webpack&utm_medium=banner&utm_campaign=sponsorship"
-      >
-        <img
-          src={AGLogo}
-          width={260}
-          height={140}
-          alt="ag grid"
-          loading="lazy"
-        />
-        <img
-          className="sponsors__img__webpack"
-          src={WebpackIcon}
-          alt="webpack"
-          width={150}
-          height={150}
-          loading="lazy"
-        />
-        <div className="sponsors__content-cont">
-          ag-grid is proud to partner with webpack
+      <div className="sponsors__content">
+        <div className="sponsors__link-wrapper">
+          <Link to="https://www.ag-grid.com/?utm_source=webpack&utm_medium=banner&utm_campaign=sponsorship">
+            <img
+              className="agGridLogo"
+              src={AGLogo}
+              alt="ag grid"
+              width={220}
+              loading="lazy"
+            />
+          </Link>
+        </div>
+        <div className="sponsors__link-wrapper">
+          <Link to="https://charts.ag-grid.com/?utm_source=webpack&utm_medium=banner&utm_campaign=sponsorship">
+            <img
+              className="agChartsLogo"
+              src={AGChartsLogo}
+              alt="ag charts"
+              width={220}
+              loading="lazy"
+            />
+          </Link>
         </div>
-      </Link>
+        <div className="sponsors__link-wrapper-2">
+          <Link to="https://www.ag-grid.com/?utm_source=webpack&utm_medium=banner&utm_campaign=sponsorship">
+            <div className="sponsors__tagline">
+              Datagrid and Charting for Enterprise Applications
+              <br />
+              <span style={{ color: '#0070E0', fontSize: '0.70em' }}>
+                Learn more...
+              </span>
```

Comment:
Do we need this `learn more...` text as the entire section is already clickable including the logos?

Reply:
Thanks, I've removed this